static-libraries/js/khan-lite.js

```diff
@@ -251,6 +251,11 @@ function clear_messages(msg_type) {
     return $("#message_container");
 }
 
+// To clear all messages including django and ajax.
+function clearAllMessages(msg_type) {
```

Comment:
The argument has no purpose at all. The function should clear django messages or normal messages loaded in #message_container.